projects/view/client/rtProject/rtProjectProfile.css

```diff
@@ -0,0 +1,70 @@
+/*  COLUMN SETUP  */
```

Comment:
In this, and all, file(s), use two spaces per level of indentation. Convert tabs to spaces.